scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamOp.scala

```diff
@@ -57,14 +68,17 @@ object BeamOp extends Serializable {
           .apply(MapElements.via(new SimpleFunction[KV[K, V], KV[K, java.lang.Iterable[U]]]() {
             override def apply(input: KV[K, V]): KV[K, lang.Iterable[U]] =
               KV.of(input.getKey, Seq(input.getValue.asInstanceOf[U]).toIterable.asJava)
-          })).setCoder(KvCoder.of(OrderedSerializationCoder(ordK, kryoCoder), IterableCoder.of(kryoCoder)))
+          })).setCoder(
+          KvCoder.of(OrderedSerializationCoder(ordK, kryoCoder), IterableCoder.of(kryoCoder))
+        )
       case notComposedOrSum =>
         pcoll.apply(ParDo.of(
           MapFn[KV[K, java.lang.Iterable[V]], KV[K, java.lang.Iterable[U]]] { elem =>
             KV.of(
               elem.getKey,
               notComposedOrSum(elem.getKey, elem.getValue.asScala.toIterator).toIterable.asJava)
-          })).setCoder(KvCoder.of(OrderedSerializationCoder(ordK, kryoCoder), IterableCoder.of(kryoCoder)))
+          })).setCoder(
+          KvCoder.of(OrderedSerializationCoder(ordK, kryoCoder), IterableCoder.of(kryoCoder)))
```

Comment:
I think it should be only one (maybe two) place/s where we do creation of `OrderedSerializationCoder`